### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: ziti/config/config.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	return &Config{
+		ZtAPI:    ztApi,
+		ID:       idConfig,
+	}
+}
+
+func NewFromFile(confFile string) (*Config, error) {
+	conf, err := ioutil.ReadFile(confFile)
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("Config file (%s) is not found ", confFile))
```
Comment (dovholuknf) [inline ziti/config/config.go]: this isn't a log but i could see it getting logged elsewhere. should errors follow the lowercase letters a the start rule?

File: ziti/config/config.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	}
+
+	c := Config{}
+	err = json.Unmarshal(conf, &c)
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("Failed to load ZT configuration (%s): %v", confFile, err))
```
Comment (dovholuknf) [inline ziti/config/config.go]: same lower/capital case comment - i won't make others but there might be more to resolve.

File: ziti/config/token.go [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+		delete(mapClaims, "StandardClaims")
+
+		if stdMap, ok := std.(map[string]interface{}); ok {
+			for k, v := range stdMap {
+				if !isZeroValue(v) {
+					mapClaims[k] = v
+				}
+			}
+		} else {
+			return nil, errors.New("could not converted standard claims section to map")
```
Comment (dovholuknf) [inline ziti/config/token.go]: "could not converted" - typo?

File: ziti/config/config.go (hunk 2 above)
Comment (michaelquigley) [reply ziti/config/config.go]: we also shouldn't be using `ZT`

File: ziti/edge/conn_test.go [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+package edge
+
+import (
+	"testing"
+)
+
+func Test_edgeChannel_Host(t *testing.T) {
+
+}
```
Comment (michaelquigley) [inline ziti/edge/conn_test.go]: Do we need to keep this around if it doesn't do anything?

File: ziti/config/config.go (hunk 1 above)
Comment (michaelquigley) [reply ziti/config/config.go]: Can also use `fmt.Errorf` instead of `errors.New(fmt.Sprintf(...`

File: ziti/config/config.go (hunk 2 above)
Comment (plorenz) [reply ziti/config/config.go]: fixed

File: ziti/config/config.go (hunk 1 above)
Comment (plorenz) [reply ziti/config/config.go]: fixed. Switched to using errors.Errorf where appropriate and fixed casing where I found it.

File: ziti/config/token.go (hunk 3 above)
Comment (plorenz) [reply ziti/config/token.go]: fixed

File: ziti/edge/conn_test.go (hunk 4 above)
Comment (plorenz) [reply ziti/edge/conn_test.go]: fixed

**Pull request merged**